tasks/connectors/edgescan/lib/edgescan_definition.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module Kenna
+  module Toolkit
+    module Edgescan
+      class EdgescanDefinition
+        attr_accessor :data
+
+        def initialize(definition)
+          @data = definition
+        end
+
+        def to_kenna_definition
+          {
+            "scanner_type" => "Edgescan",
+            "scanner_identifier" => data["id"],
+            "name" => data["name"],
+            "description" => data["description_src"],
+            "solution" => data["remediation_src"],
+            "cve_identifiers" => cves,
+            "cwe_identifiers" => cwes
+          }.compact
+        end
+
+        private
+
+        def cves
```

Comment:
It's better to set a precedence for the identifiers and send in only the one you want used. So if cve send cve, only send cwe if no cve. Make sense?

Reply:
Makes perfect sense. I made that change in this commit: https://github.com/KennaSecurity/toolkit/pull/170/commits/4da1aab47d8a0dd8a7b7d62516556fde143b27b7

Closing this for now but feel free to re-open if you think I should have gone another route